src/app/task.service.ts

```diff
@@ -14,8 +14,8 @@ export class TaskService {
 
   taskList$: Observable<TaskData[]> = this.taskListSubject.asObservable();
 
+  url = environment.apiUrl;
```

Comment:
you can directly use the environment.apiUrl otherwise you'll need to declare a new variable for each service that you then use